telemetry/packages/fake/src/index.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Main file which initialises the generation of the data series and uploads to GUI in real time.
+ * @param runTime (CLI) simulation time in ms (not real time, based on sensor timesteps)
```

Comment:
Doesn't look like this CLI argument is ever used? I assume it was replaced with the `trackLength` limit instead?